README.md

```diff
@@ -112,7 +112,6 @@ accelerate launch --config_file=recipes/accelerate_configs/zero3.yaml src/open_r
     --dataset_name open-r1/OpenR1-Math-220k \
     --learning_rate 5.0e-5 \
     --num_train_epochs 1 \
-    --packing \
```

Comment:
Removing packing as the default recommendation since we found a slight performance degradation with reasoning models (data gets split across chunk boundaries)